src/bundler.ts

```diff
@@ -243,7 +276,7 @@ class Bundler {
 
   // TODO(usergenic): Migrate "Old Polymer" detection to polymer-analyzer with
   // deprecated feature scanners.
-  oldPolymerCheck(analyzedRoot: Documents) {
+  oldPolymerCheck(analyzedRoot: Document) {
```

Comment:
I'll leave this here for this PR but we can remove after we have a version supported/unsupported story nailed down.